lib/developer_portal/lib/liquid/tags/active_docs.rb

```diff
@@ -43,6 +45,9 @@ def version
         end
       end
 
+      def swagger2?
+        @version > "1.2"
```

Comment:
we probably would need to make sure if there's anyone still using version < 2.

Reply:
How do we do that?

Reply:
It should be part of deprecation plan for the road to OAS 3?
/cc @3scale/product

Reply:
@josemigallas In case that we need to do that, you will need to connect to the preview DB going to [the deploy repository](https://github.com/3scale/deploy) locally and doing:
```
./bin/remote system preview01 ssh // connect to preview
RAILS_ENV=preview bundle exec rails console // use rails console inside preview
```
And then you can do: `ApiDocs::Service.where.has { swagger_version < '2.0' }.count` to see how many we have 😄 And this is the result at the moment:
![image](https://user-images.githubusercontent.com/11318903/53007966-9d193a80-3438-11e9-9a62-d7efc6d19935.png)

Reply:
But this is the amount of services, not the amount of accounts, which will be less but still...